docs/UserGuide.md

```diff
@@ -1,19 +1,51 @@
-# User Guide
+# WerkIt! User Guide
 
 ## Introduction
 
 {Give a product intro}
 
+---
+
 ## Quick Start
 
 {Give steps to get started quickly}
 
 1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+2. Down the latest version of WerkIt! from [here (WIP)](http://link.to/duke).
```

Comment:
Download instead of "Down"?

Reply:
Oops, missed that part. Thanks for spotting it :)

Reply:
Fixed in Commit babb92d